embedding/csharp/Tizen.Flutter.Embedding/Tizen.Flutter.Embedding.csproj

```diff
@@ -5,8 +5,16 @@
     <TargetFramework>netstandard2.0</TargetFramework>
   </PropertyGroup>
 
-  <ItemGroup>
+  <PropertyGroup>
+    <TizenApiLevel Condition="'$(TizenApiLevel)' == ''">4</TizenApiLevel>
```

Comment:
Tip: This default value may temporarily be changed to 9 during development (to enable IntelliSense).